openapi.yml

```diff
@@ -3384,6 +3387,96 @@ paths:
+          description: 'Successful response returns the current number of signins, expected invites and expected invites for the next N number of hours.'
+          content:
+            application/json:
+              schema:
+                $ref: ./models/CapacityResponse.v1.yaml
+              examples:
+                default:
+                  value:
+                    signins_count: 189
+                    invites_expected_next_hour: 1
+                    invites_by_hour:
+                      - range_start: '2020-07-17T00:59:59.999Z'
+                        range_end: '2020-07-17T01:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T01:59:59.999Z'
+                        range_end: '2020-07-17T02:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T02:59:59.999Z'
+                        range_end: '2020-07-17T03:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T03:59:59.999Z'
+                        range_end: '2020-07-17T04:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T04:59:59.999Z'
+                        range_end: '2020-07-17T05:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T05:59:59.999Z'
+                        range_end: '2020-07-17T06:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T06:59:59.999Z'
+                        range_end: '2020-07-17T07:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T07:59:59.999Z'
+                        range_end: '2020-07-17T08:59:59.999Z'
+                        expected_visitors: 1
+        '400':
+          description: A generic error
+          content:
+            application/json:
+              schema:
+                $ref: ./models/ErrorsList.v1.yaml
+              examples:
+                default:
+                  value:
+                    errors:
+                      - domain: some text
+                        attribute: some text
+                        code: some text
+                        message: some text
+                      - domain: some text
+                        attribute: some text
+                        code: some text
+                        message: some text
+        '401':
+          description: "You don't have permission to view the details of the location's capacity."
+        '403':
+          description: You do not have permission for this action
+      operationId: getLocationCapacity
+      description: Gets the details of the capacity in a location.
+      parameters:
+        - schema:
+            type: string
+          in: query
+          name: local_time
+          description: Current local time with offset value.
```

Comment:
```suggestion
        - schema:
            type: string
            format: date-time
          in: query
          name: start_time
          description: ISO8601 timestamp to use as the start point for the capacity estimate report. Defaults to the current local timestamp of the location if not provided
```

Reply:
Ok. Now all calculations will be done based on the location's timezone from db, I removed the local_time from the query params. We can find the current time from the location's timezone.

Reply:
Looking at the scope for this quarter, we decided to calculate data based on UTC(just the way data is saved inside db). Because current_time is just the time now, no matter whether in UTC or in PST. And we calculate the N number of hours from the current time. So 2.30 PM - 3.30 PM etc for N number of times.